packages/ui/src/composites/wui-transaction-list-item/index.ts

```diff
@@ -0,0 +1,81 @@
+import { html, LitElement } from 'lit'
+import { property } from 'lit/decorators.js'
+import '../../components/wui-text/index.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import '../wui-transaction-visual/index.js'
+import styles from './styles.js'
+import { type TransactionType, TransactionTypePastTense } from '../../utils/TypeUtil.js'
+import { type TransactionStatus, type TransactionDirection } from '@web3modal/common'
+
+@customElement('wui-transaction-list-item')
+export class WuiTransactionListItem extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
```

Comment:
Same as above (and I guess for any other cases where style array is present) verify that all of these are necessary